90-ybacklight.rules

```diff
@@ -0,0 +1,2 @@
+ACTION=="add", SUBSYSTEM=="backlight", RUN+="/bin/chgrp video /sys/class/backlight/%k/brightness"
+ACTION=="add", SUBSYSTEM=="backlight", RUN+="/bin/chmod g+w /sys/class/backlight/%k/brightness"
```

Comment:
`… GROUP="video", MODE="0660"`?

Reply:
I'm also not a huge fan of this way of doing it but I couldnt find any other that worked out, also found cases of it here https://bbs.archlinux.org/viewtopic.php?id=260428
even if it did I think that would change all the files from subsystem backlight(?) which might be a bit overkill.

as much as I'd like it to work it doesnt seem to when the rule is
```
 ACTION=="add", SUBSYSTEM=="backlight",  GROUP="video", MODE="0660"
```

unless this isnt what you meant:
```
[cat:~]$ cat /usr/lib/udev/rules.d/90-ybacklight.rules
ACTION=="add", SUBSYSTEM=="backlight", GROUP='video", MODE="0660"
[cat:~]$ ls -l /sys/*/backlight 
total 0
lrwxrwxrwx 1 root root 0 Aug 21 15:00 acpi_video0 -> ../../devices/pci0000:00/0000:00:02.0/backlight/acpi_video0
lrwxrwxrwx 1 root root 0 Aug 21 15:00 intel_backlight -> ../../devices/pci0000:00/0000:00:02.0/drm/card0/card0-LVDS-1/intel_backlight
```

Reply:
nah, nevermind, thanks for the PR :)